src/sync.rs

```diff
@@ -167,6 +167,20 @@ impl<T> GILOnceCell<T> {
         *inner = Some(value);
         Ok(())
     }
+
+    /// Takes the value out of the cell, moving it back to an uninitialized state.
+    ///
+    /// Has no effect and returns None if the cell has not yet been written.
+    pub fn take(&mut self) -> Option<T> {
+        self.0.get_mut().take()
```

Comment:
Could we avoid the unsafe block `GILOnceCell::get_mut` in the same manner, i.e. `self.0.get_mut().get_mut()`?

Reply:
Done.